libs/e2e/src/util/BoxChart.ts

```diff
@@ -59,10 +59,13 @@ export class BoxChart extends Chart<IBox> {
       : { ...boxCoordinate };
   };
 
-  private readonly getBoxCoordinate = (
+  private readonly getD = (
```

Comment:
for my learning, what is this `D`?

Reply:
The d value of svg element.
This is a extra function added to address cyclone complexity.
Seems git hub did not recognize it correctly.
getBoxCoordinate is still below